lib/krb5/init_creds_pw.c

```diff
@@ -3657,7 +3657,8 @@ KRB5_LIB_FUNCTION void KRB5_LIB_CALL
 krb5_init_creds_free(krb5_context context,
 		     krb5_init_creds_context ctx)
 {
-    free_init_creds_ctx(context, ctx);
+    if (ctx)
+        free_init_creds_ctx(context, ctx);
     free(ctx);
 }
 
```

Comment:
I know free(NULL) is OK but, maybe put both inside the branch?

Reply:
Or not.

Reply:
It's my style.